quiver/run-quiver.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
Let's think about how the install command will look. Right now, the command to install cloud is:
```bash
curl https://raw.githubusercontent.com/uProxy/uproxy-docker/master/install-cloud.sh | sh
```

I think you could move this to the top-level directory of the repo and rename it `install-quiver.sh`. Then we can tell users to install by:
```bash
curl https://raw.githubusercontent.com/uProxy/uproxy-docker/master/install-quiver.sh | sh
```
WDYT?